### DISCUSSION TITLE
WIP limits?

### DISCUSSION THREAD

**Discussion by tomdavidson** [makeplane/plane#3243] [General]
Hello, I am just poking around Plane and it seems quite promising but an extremely important functionality for is WIP limits. Does the design of Plane facilitate implementing WIP limits or would it be an awkward bolt-on?

**Comment by tomdavidson** [comment]
- https://kanbantool.com/kanban-library/why-kanban/why-limit-your-wip
- https://businessmap.io/kanban-resources/getting-started/what-is-wip
- https://www.geeksforgeeks.org/software-engineering/benefits-of-wip-limits-in-kanban/

**Comment by conny3496** [comment]
I agree. As WIP-limits are totally part of Kanban and the vision of Taiichi Ōno.
There's an open feature request you can follow: https://github.com/makeplane/plane/issues/5129

**Reply by tomdavidson** [reply]
Perfect !
(and sorry I didnt search for the issue again before my last comment)
